help/restrict-direct-messages.md

```diff
@@ -0,0 +1,38 @@
+# Restrict direct messages
+
+{!admin-only.md!}
+
+In Zulip, users can exchange direct messages with other users,
```

Comment:
This can be a follow-up commit, since it's not directly related to the renaming, but should "direct messages" here be a link to the main article on them?

(Or should we have a "See also" on this page).